msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -260,13 +260,34 @@ To share the entire app to stage, in the app manifest, you must configure `meeti
 > [!NOTE]
 > For your app to pass validation, when you create a deep link from your website, web app, or Adaptive Card, use **Share in meeting** as the string or copy.
 
+## Build in-meeting document signing app
```

Comment:
Potentially unrelated to this review, but when I open the review site I do not see this page in the left hand table of contents. Is that expected or is there TOC broken?
![image](https://user-images.githubusercontent.com/6508687/197222668-a50acfd0-3a2a-400f-af1d-1f2e91f8e1bf.png)